packages/flutter_test/test/window_test.dart

```diff
@@ -5,7 +5,7 @@
 import 'dart:ui' as ui show window;
 import 'dart:ui' show Size, Locale, WindowPadding, AccessibilityFeatures, Brightness;
 
-import 'package:flutter/widgets.dart' show WidgetsBinding;
+import 'package:flutter/widgets.dart';// show WidgetsBinding, WidgetsBindingObserver;
```

Comment:
remove the commented out code?

Reply:
Done.  Thanks: I didn't mean to leave it commented out, actually.